app/javascript/react/screens/App/Overview/Overview.js

```diff
@@ -176,8 +186,11 @@ class Overview extends React.Component {
 
     createTransformationPlanRequestAction(url).then(() => {
       retryMigrationAction(planId);
-      setMigrationsFilterAction('Migration Plans in Progress');
-      fetchTransformationPlansAction(fetchTransformationPlansUrl);
+      setMigrationsFilterAction('In Progress Plans');
```

Comment:
Looking at these again... it would be nice to break the "In Progress Plans", "Migration Plans in Progress", "Migration Plans Not Started" strings out into constants across the board. It seems like "In Progress Plans" is the same as "Migration Plans in Progress" ?